spec/tic_tac_toe_spec.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+require  './lib/player'
+require  './lib/board'
+
+player1 = 'Lu'
+RSpec.describe Player do
+  let(:player) { Player.new(player1, 'X') }
+
```

Comment:
I think player1 = 'Lu' should be inside, as you did for the player. Only the sequence actually matter.
```
let(:player1) { 'Lu' }
let(:player) { Player.new(player1, 'X') }
```
It worked on my side. Let me know if you have different point of view.